src/components/BPageNav/BPageNav.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <nav>
+      <ul role="menu" aria-label="Page Navigation">
+        <li v-for="(link, index) in links" role ="menuitem" :key="index">
+          <a
+            :href="`#${link.href}`"
+            :aria-label="`${link.label} ${link.completed ? 'Completed' : 'Uncompleted'}`"
+            @click.prevent.stop="jumpTo(link)"
+            @keyup.enter="jumpTo(link)"
+            @keyup.space="jumpTo(link)"
+          >
+            <i v-if="link.completed" class="icon-complete ion-ios-checkmark" aria-hidden="true" />
+            <i v-else class="icon-incomplete ion-record" aria-hidden="true" />
+            {{ link.label }}
+          </a>
+        </li>
+      </ul>
+    </nav>
+</template>
+
+<script>
+import events from '../../event-bus'
+
+export default {
+  name: 'b-page-nav',
+  props: {
+    //Required props
+    links: {
```

Comment:
Need to document `label` and `href` for styleguide

Reply:
Documented now